cli/azd/.golangci.yaml

```diff
@@ -12,4 +13,8 @@ linters-settings:
   lll:
     # GitHub recommended for code review
     line-length: 125
-    tab-width: 4
\ No newline at end of file
+    tab-width: 4
+  revive:
+    rules:
+      - name: unused-parameter
```

Comment:
This rule will check the unused, @ellismg

Reply:
I suspect the reason this flags additional locations that `gopls` doesn't is because these parameters are there to implement interface methods.  Part of me feels that instead of buying into this one, we should just get on a `gopls` plan since that this more official?